dashboard/test/ui/features/dance/social_share.feature

```diff
@@ -0,0 +1,62 @@
+# Testing that the social share buttons are shown/hidden in the correct scenarios
+# in Dance Party.
+
+Feature: Social share buttons
+  Scenario: Signed in 13+ user sees social share buttons
+    Given I create a teacher named "Hermoine"
+    And I am on "http://studio.code.org/s/dance/stage/1/puzzle/13"
```

Comment:
what's the best way to check that this line will be okay in all environments? i recall trying to use a script that isn't available in either staging or test in the past, and that causing the test to break in that environment

Reply:
If it's in this list it'll be available for UI tests in all environments:

https://github.com/code-dot-org/code-dot-org/blob/2935aba881ca86908188913acf05e682a778e0c1/dashboard/lib/tasks/seed.rake#L21